README.md

```diff
@@ -19,3 +19,13 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+We should start with having an idea of what the theoretical perfect pivot for quicksort would be. The theoretical perfect pivot for quicksort is the median of the array. This is because it would create two partitions of n/2. So, a good pivot would be a number that would be at least close to these partitions of n/ 2. This means that although there is only one perfect pivot in a random array, there could be several good pivots in that same array. 
+I will analyze this from the perspective of which algorithm is more likely to select the perfect pivot inside a random array. I will justify this by stating that if this method is more likely to catch, the perfect pivot is also more likely to catch a good pivot. Let's start with the baseline of picking the first number in the array as the pivot. This number has a 1/n chance of being the perfect pivot inside a random array. 
+The Median-of-three method of picking a pivot inspects the first, middle, and last elements of the part of the array under consideration and chooses the median value. This number has a 3/n chance of being the perfect pivot because it inspects three values. 
+This means that The Median-of-three method is 3 times more likely to Pick a good pivot than the first number method.
```

Comment:
That can't be right. We said that choosing a single value has a 50% chance of getting a good pivot, so this would be 150%, which is impossible.